mec/beamline.py

```diff
@@ -69,15 +70,16 @@ def lights_off():
 
 
 with safe_load('YAG screens'): # Also known as PIMs
-    from pcdsdevices.pim import PIM
-    from pcdsdevices.pim import PIMMotor
-    #mec_yag0 = mec_pim1
-    mec_yag0 = PIM('HXX:UM6:MMS:08', prefix_det='HXX:UM6:CVV:01', name='mec yag0')# ==works
-    mec_yag1 = PIM('MEC:HXM:MMS:16', prefix_det='MEC:HXM:CVV:01', name='mec yag1')# ==works
-    mec_yag2 = PIMMotor('MEC:XT2:MMS:13', name='mec yag2')
-    #mec_yag2 = PIM('MEC:XT2:MMS:13', prefix_det='MEC:XT2:CVV:01', name='mec yag2')# !=works
-    mec_yag3 = PIMMotor('MEC:XT2:MMS:29', name='mec yag3')
-    #mec_yag3 = PIM('MEC:XT2:MMS:29', prefix_det='MEC:XT2:CVV:02', name='mec yag3')# !=works
+    from pcdsdevices.pim import PIM, PIMY
+    mec_yag0 = PIMY('HXX:HXM:PIM', name='mec yag0')
+    yag0 = mec_yag0
```

Comment:
This is neat! Does it go looking in the camviewer.cfg to find the PV?

Reply:
No, that would be cool though. The PIMY class omits the camera, and just gives you control of the stage. The camera IOC on this PIM is not compatible with the normal PIM class. I don't think MEC uses the PIM camera functionality in python anyway, so I think this is OK for now until MEC's YAGs get a bit more standardized.